MessagePack/Source/MessagePack.swift

```diff
@@ -616,6 +630,12 @@ extension MessagePackValue {
     /// The contained string if `.String`, `nil` otherwise.
     public var stringValue: Swift.String? {
         switch self {
+        case .Binary(let data):
+            if let string = NSString(data: data, encoding:NSASCIIStringEncoding) {
```

Comment:
Yes, you're right - well actually the user should decide for his / her environment, but UTF-8 would be the best default setting.